common_benchmark_suite/openxla/benchmark/models/tf/bert/bert_model.py

```diff
@@ -51,6 +51,14 @@ def forward(self, inputs: Tuple[Any, ...]) -> Tuple[Any, ...]:
                         training=False).last_hidden_state
     return (output,)
 
+  @tf.function(jit_compile=True)
+  def forward_sm(self, input_ids, attention_mask):
```

Comment:
If TF doesn't accept a tuple as function input, we should refactor the model interface. Can you add a TODO and I will clean up this.

Reply:
Added https://github.com/openxla/openxla-benchmark/issues/60 and a TODO.